<?xml version="1.0"?>
<launch>
	<group ns="big_robot">
		<node pkg="gobot_software" type="command_system" name ="command_system" output="screen" required="true"/>
		<node pkg="gobot_software" type="ping_server.py" name="ping_server" output="screen" required="true"/>
	</group>
</launch>
